lib/common.js

```diff
@@ -223,6 +271,11 @@ module.exports = (config) => {
   config.utils.setSocketUpdateRate._help = `setSocketUpdateRate(value) Sets socket update rate (in ms)`
   config.utils.getSocketUpdateRate._help = `getSocketUpdateRate() Returns current socket update rate`
   config.utils.setShardName._help = `setShardName(value) Sets the shard name`
+  config.utils.getCPULimit._help = `getCPULimit(username)`
+  config.utils.setCPULimit._help = `setCPULimit(username, value)`
+  config.utils.enableGCLToCPU._help = `enableGCLToCPU(maxCPU, baseCPU, stepCPU) changes running state and does not persist between server restarts`
```

Comment:
Help text should start with a capital letter, should also actually say what the function does (even if the function name should be self-explanatory, see set/getSocketUpdateRate and others above.)

Reply:
fixed